packages/react/lib/Element/index.tsx

```diff
@@ -139,58 +139,73 @@ const Element = forwardRef<ElementRef, ElementProps>(({
             className
           )}
         >
-          { component?.hasCustomInnerContent ? rendered : component ? (
-            <div
-              className={classNames(
-                'inner oak-flex oak-gap-2 oak-p-4 oak-items-stretch',
-                depth % 2 === 0 ? 'even' : 'odd',
-              )}
-            >
-              <Icon
-                children={typeof component?.icon === 'function'
-                  ? component.icon.bind(null, component)
-                  : component?.icon}
-              />
+          { component?.hasCustomInnerContent && !element.collapsed
+            ? rendered : component ? (
               <div
                 className={classNames(
-                  'element-info oak-flex-auto oak-flex oak-flex-col oak-gap-2',
-                  'oak-justify-between'
+                  'inner oak-flex oak-gap-2 oak-p-4 oak-items-stretch',
+                  depth % 2 === 0 ? 'even' : 'odd',
                 )}
               >
-                <h6 className="junipero oak-m-0">
-                  <Text>{ component?.name as string }</Text>
-                </h6>
-                { rendered && (
-                  <div className="element-content oak-flex-auto">
-                    { rendered }
+                <Icon
+                  children={typeof component?.icon === 'function'
+                    ? component.icon.bind(null, component)
+                    : component?.icon}
+                />
+                <div
+                  className={classNames(
+                    'element-info oak-flex-auto oak-flex oak-flex-col',
+                    'oak-gap-2 oak-justify-between'
+                  )}
+                >
+                  <div>
```

Comment:
maybe we can make this div clickable to expand when clicking on the collapsed element? I was naturally expecting it to work that way when I tried it 😂